jmespath-core/src/main/java/io/burt/jmespath/parser/ExpressionParser.java

```diff
@@ -85,7 +85,7 @@ private String identifierToString(JmesPathParser.IdentifierContext ctx) {
   private void checkForUnescapedBackticks(Token token) {
     int unescapedBacktickIndex = indexOfUnescapedBacktick(token.getText());
     if (unescapedBacktickIndex > -1) {
-      errors.parseError("unexpected `", token.getLine(), token.getStartIndex() + unescapedBacktickIndex);
+      errors.parseError("syntax error unexpected `", token.getLine(), token.getStartIndex() + unescapedBacktickIndex);
```

Comment:
I agree, I wanted a ":" but there's already one before this substring. Entirely pleased with the error syntax I am not.